SPECS/kernel/config_aarch64

```diff
@@ -218,6 +218,7 @@ CONFIG_BPF_SYSCALL=y
 CONFIG_ARCH_WANT_DEFAULT_BPF_JIT=y
 CONFIG_BPF_JIT_ALWAYS_ON=y
 CONFIG_BPF_JIT_DEFAULT_ON=y
+# CONFIG_BPF_UNPRIV_DEFAULT_OFF is not set
```

Comment:
Recommend setting to `=y` since we want to disable unprivileged bpf calls by default.